extensions/cornerstone-3d/src/services/ViewportService/Cornerstone3DViewportService.ts

```diff
@@ -489,6 +502,35 @@ class Cornerstone3DViewportService implements IViewportService {
     }
   }
 
+  _getSlabThickness(displaySetOptions, volumeId) {
+    const { blendMode } = displaySetOptions;
+    if (blendMode === undefined || !displaySetOptions.slabThickness) {
+      return;
+    }
+
+    // if there is a slabThickness set as a number then use it
+    if (
+      displaySetOptions.slabThickness &&
+      typeof displaySetOptions.slabThickness === 'number'
+    ) {
+      return displaySetOptions.slabThickness;
+    }
+
+    if (displaySetOptions.slabThickness === 'auto') {
```

Comment:
Shouldn't it be something like "fullBody", "fullVolume" or "max"? "auto" is a bit vague.